mcs/class/Mono.Debugger.Soft/Test/dtest.cs

```diff
@@ -4396,5 +4396,17 @@ public void Hash ()
 		AssertValue (3.0, f);
 
 	}
+
+	[Test]
+    public void InvokeGenericMethod () {
```

Comment:
Can we put this test in IL2CPP as well?

Reply:
yes, but I think the test would fail until these changes make it into the IL2CPP repo

Reply:
That makes sense. Can you land these changes in the IL2CPP repo as well?

Reply:
sure.  One thing to note is that the IL2CPP debugger did not have this problem and dealt with this situation fairly gracefully.

Reply:
Ok. At least we should add the test so that we don't regress in this case.